test/robustness/CMakeLists.txt

```diff
@@ -5,6 +5,37 @@
 # Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 
+
+function(boost_geometry_add_robustness_test item)
```

Comment:
The robustness tests were compiled and run with ctest.
For the robustness tests that makes no sense and costs (depending on the options) a lot of time.
Therefore it's now a separate target, and not added to the test framework.